tests/phpunit/FunctionalTest.php

```diff
@@ -48,6 +62,19 @@ public function setUp()
 
     public function testRun()
     {
+        $this->assertFalse($this->writer->tableExists('simple'));
+        $this->assertFalse($this->writer->tableExists('special'));
+
+        foreach ($this->config['parameters']['tables'] as $table) {
+            if ($table['dbName'] === 'simple') {
+                $this->assertTrue($table['incremental']);
```

Comment:
to mi přijde takový zvláštní si testovat parametry jestli sis je připravil správně:)

Reply:
Ten swap je jen pri `writeFullFromS3`.
Tohle jsem dal jen jako kontrolu do budoucna kdyby nekdo zmenil tu testovaci konfiguraci, ktera se pouziva i v jinych testech.

Reply:
ok